dashboard/config/application.rb

```diff
@@ -100,8 +100,6 @@ class Application < Rails::Application
       jquery.handsontable.full.css
       jquery.handsontable.full.js
     )
-    config.react.variant = :development
```

Comment:
Does this mean we're always going to be in development mode now? I don't know that any part of our process that sets NODE_ENV. Now that I think about it, I'm also not sure how we handle this in apps.

Looking into apps a little bit more, I think we now depend on babelify to handle our jsx instead of reactify (though reactify is still in apps's package.json, I think it can be removed). I wonder if we should use the same here to get them closer to the same.